magento/magento2
# SEO: Do not follow links on filter options

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/130


### Description (*)
This PR adds rel="nofollow" tag to filter option links and swatches links. We don't want bots following these links, as there's almost infinite amount of combinations filters can be set on large category page. All these pages have canonical tag pointing back to parent page anyway and won't be indexed as separate pages.

### Fixed Issues (if relevant)
Fixes issue where Googlebot hammers filter option links, causing lot of load on the server, and Googlebot using it's indexing quota on filtering links.

### Manual testing scenarios (*)
1. Open category page with products having filter options
2. Check that options have rel="nofollow"
